Integrations/integration-Symantec_Site_Review.yml

```
@@ -0,0 +1,205 @@
+            'ContentsFormat': formats['text'], # json, markdown, file, text
+            'Contents': message
+        })
+        sys.exit(0)
+
+    def http_request(method, url, body=None):
+
+        headers = {
+            'Content-Type': 'application/json',
+            'Accept': 'application/json'
+        }
+
+        res = requests.request(method, url, verify=VERIFY_CERTIFICATE, headers=headers, data=body)
+
+        if res.status_code < 200 or res.status_code >= 300:
+            return_error_and_exit('Failed to retrieve data.\nURL: {}, Status Code: {}, Response: {}'.format(url, res.status_code, res.text))
+
+        return res.json()
+
+    """
+    Symantec Site Review Commands
+    """
+
+    def list_risk_levels():
+        full_url = BASE_URL + '/resource/risklevels'
+        res = http_request('GET', full_url)
+        return res
+
+    def list_risk_levels_command():
+        raw_list = list_risk_levels()
+        #demisto.log(raw_list)
+        risk_content = []
+        for level in raw_list:
+            content = {
+                "Risk Level": level['number'],
+                "Description": level['name']
+            }
+            risk_content.append(content)
+
+        md = tableToMarkdown('Threat Risk Level Descriptions', risk_content, ['Risk Level', 'Description'])
+
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['json'],
+            'Contents': risk_content,
+            'HumanReadable': md,
+            'EntryContext': {
+                'Symantec.Site.RiskLevels': risk_content
+            }
+        })
+
+    def site_lookup(site):
+        #Setting an empty 'captcha' field is required to bypass the ui captcha requirement
+        req_body = {}
+        req_body['url'] = site
+        req_body['captcha'] = ""
+
+        full_url = BASE_URL + '/resource/lookup'
+        res = http_request('POST', full_url, json.dumps(req_body))
+
+        return res
+
+    def site_lookup_command():
+        """
+        urls argument can support a list separated by comma
+        """
+        sites = demisto.args().get('urls')
+        site_details = []
+        if isinstance(sites, basestring):
+            sites = sites.split(',')
+        for site in sites:
+            raw_lookup = site_lookup(site)
+            site_detail = {}
+            md_headers = ['URL','CategorizationID','CategorizationName']
+            site_detail['URL'] = raw_lookup['url']
+            site_detail['CategorizationID'] = raw_lookup['categorization'][0]['num']
+            site_detail['CategorizationName'] = raw_lookup['categorization'][0]['name']
+            if raw_lookup['resolvedDetail']['ipAddress']:
+                site_detail['ResolvedIP'] = raw_lookup['resolvedDetail']['ipAddress']
+                md_headers.append('ResolvedIP')
+            if raw_lookup['threatriskLevel']:
+                site_detail['ThreatRiskLevel'] = raw_lookup['threatriskLevel']
```

This way you will not need the else condition
```suggestion
                site_detail['ThreatRiskLevel'] = raw_lookup.get('threatriskLevel', '')
```